Dockerfile.dapper

```diff
@@ -29,7 +29,9 @@ RUN mkdir -p /go/src/golang.org/x && \
 RUN rm -rf /go/src /go/pkg
 
 ARG DAPPER_HOST_ARCH
-ENV ARCH $DAPPER_HOST_ARCH
+ENV ARCH=$DAPPER_HOST_ARCH
+ENV GOARCH=$GOARCH
+ENV GOARM=$GOARM
```

Comment:
if we are going to expand values and set as env we should probably declare them explicitly as build ARGs above here